src/base/preferences.cpp

```diff
@@ -463,6 +463,28 @@ void Preferences::setServerDomains(const QString &str)
     setValue("Preferences/WebUI/ServerDomains", str);
 }
 
+QHostAddress Preferences::getWebUiAddress() const
+{
+    auto addr_s = value("Preferences/WebUI/Address", "*").toString();
+    if(addr_s == "*") {
```

Comment:
`if (`

Reply:
In other places too. 
Also another unrelated comment that doesn't deserve it's own line because there is no rule for it IIRC, is that reference and pointer symbols are usually attached to the variable name in this project.